### ISSUE TITLE
Building the "Flat archives" need to be scripted.

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt-distribution#25]
Currently, after building a release, I manually build the "flat-structure" files. This should be scripted too. 

What I currently do by hand: 

 - Go into the zbuild/bolt-3.x.x` folder, and do: 

```
mv public/* public/.htaccess .
rm -rf public
nano .bolt.yml
```

In `.bolt.yml` I set: 

```
    web: .
    themebase: theme
    files: files
    view: bolt-public/view
```

 - then, `nano index.php`, change the line with the include to: 

```
$app = require __DIR__ . '/vendor/bolt/bolt/app/web.php';
```

And then to build the archives: 

```
cd ..
tar -czf ../files/bolt-3.2.4-flat-structure.tar.gz bolt-3.2.4
zip -rq --symlinks ../files/bolt-3.2.4-flat-structure.zip bolt-3.2.4
cd ..
```

**Comment by GawainLynch** [comment]
Can you bug me about this next week during down-time … I think we can include this and a few tweaks, and I wouldn't mind seeing if the archive installs can be even more novice friendly.

**Comment by GawainLynch** [comment]
Fixed in be730d1e4469facf83fb24dcbe26cc00652520f9
